bookwyrm_backend/src/main/java/com/bookwyrm/backend/book/controller/BookController.java

```diff
@@ -66,13 +75,115 @@ public ResponseEntity<BookUploadPayload> createBook(
         return ResponseEntity.status(status).body(response);
     }
 
+    //This utility function removes all articles from a list of Strings, it is not case sensitive
+    private ArrayList<String> removeArticles(List<String> unfiltered){
+        ArrayList<String> filtered = new ArrayList<String>(unfiltered);
```

Comment:
new ArrayList<String> can be replaced with new ArrayList<> to clear the warnings. In almost every case following, the comments are from warnings not heeded.